tests/spark/autologging/datasource/test_spark_datasource_autologging.py

```diff
@@ -24,6 +26,12 @@ def _get_expected_table_info_row(path, data_format, version=None):
 #   (it is not reset between tests)
 
 
+@pytest.fixture(autouse=True)
+def disable_telemetry(monkeypatch):
+    monkeypatch.setenv("MLFLOW_DISABLE_TELEMETRY", "true")
+    set_telemetry_client()
```

Comment:
I think this is a test-only issue.
I cannot repro in local or on databricks. The issue was that py4j callback server wasn't started successfully, and MLflow tries to initialize and causes error.